### DISCUSSION TITLE
What is the production release timeline for Supabase UI?

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#3333] [Questions] [answered]
Hello Supabase,

Supabase UI (ui.supabase.io) looks like a useful library, however, there is the following warning at the very beginning of the docs:

⚠️ This library is still a work in progress and is not production ready

Can you share the planned release timeline for the library?

**Comment by MildTomato** [comment, marked as answer]
Hey @general-durval 

the library is mostly stable but I’ve kept the warning message up mainly because prop names have been changing and the library is still missing basic theming functionality.

We use it ourselves at Supabase but I’d suggest waiting until theming is out so you can customise it.

I’ll also update the docs and keep them maintained, as they’ve also become out of date. 

I think sometime in Oct ‘21 is a reasonable expectation - and I’ll get a changelog on the website outlining any major changes which will help anyone using in production.
